variables.tf

```diff
@@ -1,44 +1,42 @@
-variable "service" {
-  description = "Name of service"
-}
-
-variable "name" {
-  description = "Name of secret"
-  default     = ""
+variable "automatically_after_days" {
+  description = "Number of days between automatic scheduled rotations of the secret"
+  default     = 1
 }
 
 variable "description" {
   description = "Description of the secret"
-  default     = ""
 }
 
 variable "kms_key_id" {
   description = "AWS KMS customer master key (CMK) to be used to encrypt the secret"
-  default     = ""
+  default     = null
```

Comment:
@JonRoma , why is this set to a default of null while others have no default defined? You have one other explicit default of null in here, but most don't have defaults. Was that intentional?